projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -120,4 +120,71 @@ export class NodesApi extends ApiClientFactory {
       return null;
     }
   }
+
+  /**
+   * Delete multiple nodes.
+   * @param nodeIds The list of node IDs to delete.
+   * @param permanent Delete permanently, without moving to the trashcan? (default: true)
+   */
+  async deleteNodes(nodeIds: string[], permanent: boolean = true): Promise<any> {
```

Comment:
can we raise a jira to add this support to delete multiple nodes or lock nodes in the JS-APi library? so that we don't implement that in each app, cc @MichalKinas

Reply:
Good idea, raised Jira: https://alfresco.atlassian.net/browse/ACS-5994 feel free to add some details if I missed anything